users/gurtatiLND/website/webpage.html

```diff
@@ -34,7 +34,7 @@ <h1>Tatiana Gurova</h1>
                     <h1 class="heading-home">HI, I'M TATIANA GUROVA</h1>
```

Comment:
I think this section needs the "home" id attribute so that the navigation bar works when you click on Home from lower down on the page

Reply:
yeep as well I already did it on my next PR